amaranth/build/run.py

```diff
@@ -187,6 +187,52 @@ def mkdirs(path):
 
         return RemoteSSHBuildProducts(connect_to, root)
 
+    def execute_docker(self, root="build", *, container_image, container_workdir="/amaranth_build/",
+                       docker_args=[], run_script=True):
```

Comment:
The `run_script` argument shouldn't exist (even for `execute_local`, where we should deprecate it).